### DISCUSSION TITLE
PV Ladeplanung -> Voll

### DISCUSSION THREAD

**Discussion by JauntyJosef** [wiggal/GEN24_Ladesteuerung#61] [General]
Ich habe im WebUI die Ladeplanung auf voll gesetzt. Im Log steht folgendes:

```
aktuellePrognose: 1126
RestTagesPrognose: 14409
PrognoseAbzugswert/Stunde: 950
TagesPrognose - Abzugswerte: 4518
Grundlast_Summe für Tag: 2794
aktuellePVProduktion/Watt: 515
aktuelleEinspeisung/Watt: 286
aktuelleBatteriePower/Watt: 0
GesamtverbrauchHaus/Watt: 229
aktuelleBattKapazität/Watt: 4966
Batteriestatus in Prozent: 51.5 %
LadewertGrund: Prognoseberechnung / BatSparFaktor
Bisheriger Ladewert/Watt: 0
Bisheriger Ladewert/Prozent: 0.0 %
Neuer Ladewert/Watt: 232
Neuer Ladewert/Prozent: 2.26 %
newPercent_schreiben: 0

Alte und Neue Werte unterscheiden sich weniger als die Schreibgrenzen des WR, NICHTS zu schreiben!!
```

Die Schreibgrenzen sind nicht erreicht, darum wird wohl nicht geschrieben (stehen auf 700). Aber wäre es, für den Fall, dass ich auf VOLL stelle nicht sinnvoll, dass die Ladeleistung nicht jedenfalls auf 100% gestellt wird?

**Comment by JauntyJosef** [comment]
Und wenn ich per Modbus TCP 100% in das Laderegister schreibe, dann stellt das Programm es wieder auf 0%.

```
aktuellePrognose: 1206
RestTagesPrognose: 14229
PrognoseAbzugswert/Stunde: 1350
TagesPrognose - Abzugswerte: 4428
Grundlast_Summe für Tag: 2724
aktuellePVProduktion/Watt: 528
aktuelleEinspeisung/Watt: -6
aktuelleBatteriePower/Watt: -335
GesamtverbrauchHaus/Watt: 199
aktuelleBattKapazität/Watt: 4966
Batteriestatus in Prozent: 51.5 %
LadewertGrund: Prognoseberechnung / BatSparFaktor
Bisheriger Ladewert/Watt: 10241
Bisheriger Ladewert/Prozent: 100.0 %
Neuer Ladewert/Watt: 0
Neuer Ladewert/Prozent: 0.0 %
newPercent_schreiben: 1

Am WR geschrieben: 0.0% = 0W
```

Verstehe ich da was falsch?

**Reply by wiggal** [reply]
Hi,
da scheint noch was nicht richtig zu laufen.

Wenn du "WebUI die Ladeplanung auf voll " stellst und speicherst, dann muss im Logfile nach dem nächsten Lauf folgendes stehen:
`LadewertGrund:  Manuelle Ladesteuerung in PV-Planung ausgewählt.`
Und er schreibt die Wattzahl die in der config.ini unter MaxLadung definiert ist (unabhänging von den Schreibgrenzen).

Ist die Datei `Watt_Reservierung.json` für den Apache Webserver schreibbar, hier schreibt er die Eingaben aus der "WebUI Ladeplanung" hinein. Ganz am Schluss muss für 100% folgendes stehen:
`    "ManuelleSteuerung": 1  `

**Comment by JauntyJosef** [comment]
`"ManuelleSteuerung": 1 ` steht in der Datei, aber im darauffolgenden Log steht nichts von `LadewertGrund:  Manuelle Ladesteuerung in PV-Planung ausgewählt.`.

**Reply by wiggal** [reply]
In der congig.ini die Reservierung aktiviert sein:
`PV_Reservierung_steuern = 1`

**Comment by JauntyJosef** [comment]
Ja, genau das wars! Danke sehr.
So richtig durchschaut habe ich manche Sachen noch nicht.

**Reply by wiggal** [reply]
Ja ich weiß, ist schon sehr komplex geworden, mit den ganzen Funktionen die inzwischen rein gepackt sind.

**Reply by JauntyJosef** [reply]
Es ist auch einfach so, dass es für denjenigen , der es entwickelt und sich darüber Gedanken hat, leichter ist. Für die Anderen ist die  Dokumentation dann auch oft nicht so einfach zu verstehen.

Hast du eigentlich schon einmal darüber nachgedacht MQTT zu implementieren? Dann könnte man es besser mir anderen System steuern ... nur so ein Gedanke.

**Reply by wiggal** [reply]
ne,  hab auch gar keine Erfahrung mit MQTT
